docs/src/runtime/zk-elgamal-proof.md

```diff
@@ -122,3 +122,12 @@ proofs.
     ciphertexts encrypt the same message.
   - Mathematical description and proof of security:
     [[Notes]](https://github.com/solana-labs/solana/blob/master/docs/src/runtime/zk-docs/ciphertext_ciphertext_equality.pdf)
+
+#### Ciphertext Validity proofs
+
+- `VerifyGroupedCiphertextValidity`:
+
+  - The grouped ciphertext validity proof certifies that a grouped ElGamal
+    cipehrtext is well-formed
+    - Mathematical description and proof of security:
+      [[Notes]](https://github.com/solana-labs/solana/blob/master/docs/src/runtime/zk-docs/ciphertext_validity.pdf)
```

Comment:
Should this point to agave sources? I don't know if this change will immediately get propagated to the solana repo

Reply:
Oh yes, I guess I should update the rest of the links as well.